ote_sdk/ote_sdk/tests/entities/test_datasets.py

```diff
@@ -562,10 +560,6 @@ def test_dataset_entity_append(self):
         dataset.append(item_to_add)
         expected_items.append(item_to_add)
         assert dataset._items == expected_items
-        # Checking that ValueError exception is raised when appending DatasetEntity with "media" is "None" attribute
```

Comment:
why?

Reply:
Because type annotation for DatasetItemEntity object requires “media” initialization parameter equal to IMedia2DEntity, not Optional[IMedia2DEntity]. Because of this, I can’t append DatasetItemEntity object with “media” attribute equal to “None”